bin/civihr-install

```diff
@@ -0,0 +1,820 @@
+#!/bin/bash
```

Comment:
It's an extra step, but not a required step. They can still execute it just fine without removing the extension.

The thing with the extension is just a matter of organization of the project. As I said, with the extension you don't have to open the file to figure out if it's a bash script or not. What if, in the future, we also have self executable python scripts on the same dir?

One interesting option though, is what civicrm-buildkit does. It has a `bin` dir and a `lib` dir. Both have bash scripts, but those inside `bin` are supposed to be self-executable and don't have any extension. Scripts on `lib` should only be "included" and have the `.sh` extension. Maybe we can do the same for our script (related to this, check my comment about splitting the file)